File: Modul-2/soal-5/PRAK205_2410817210017_OrlandoSugian.py

print("Input nilai A dan B :")
inputnilai = []
while(len(inputnilai) < 2):
    x = input()
    x = x.split()
    inputnilai += x
tinggia = float(inputnilai[0])
smiringb = float(inputnilai[1])
alasc = (smiringb ** 2 - tinggia ** 2) ** 0.5
kll = tinggia + smiringb + alasc
luas = 1/2 * tinggia * alasc
print(f"Alas = {alasc:.0f} cm")
print(f"Tinggi = {tinggia:.0f} cm")
print(f"Keliling = {kll:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: Modul-2/soal-5/PRAK205_2410817210017_OrlandoSugian.c
#include<stdio.h>
#include<math.h>
int main()
{
float tinggia, smiringb, kll, luas;
double alasc;
printf("Input nilai A dan B : \n");
scanf("%f %f", &tinggia, &smiringb);
alasc = sqrt ((smiringb * smiringb) - (tinggia * tinggia));
kll = tinggia + smiringb + alasc;
luas = 0.5 * tinggia * alasc;
printf("Alas = %.0f cm\n", alasc);
printf("Tinggi = %.0f cm\n", tinggia);
printf("Keliling = %.0f cm\n", kll);
printf("Luas = %.0f cm^2", luas);
return 0;
}